bench/irmin-pack/tree.ml

```diff
@@ -972,87 +977,74 @@ let suite : suite_elt list =
       speed = `Quick;
       run =
         (fun config ->
-          Bench_inodes_32.run_read_trace
-            { config with inode_config = `Entries_32; store_type = `Pack });
+          let config =
+            { config with inode_config = (32, 256); store_type = `Pack }
+          in
+          let (module Store) = store_of_config config in
+          Store.run_read_trace config);
     };
     {
       mode = `Read_trace;
       speed = `Slow;
       run =
         (fun config ->
-          Bench_inodes_32.run_read_trace
-            { config with inode_config = `Entries_32; store_type = `Pack });
+          let config =
+            { config with inode_config = (32, 256); store_type = `Pack }
+          in
+          let (module Store) = store_of_config config in
+          Store.run_read_trace config);
     };
     {
       mode = `Chains;
       speed = `Quick;
       run =
         (fun config ->
-          Bench_inodes_32.run_chains
-            { config with inode_config = `Entries_32; store_type = `Pack });
+          let config =
+            { config with inode_config = (32, 256); store_type = `Pack }
+          in
+          let (module Store) = store_of_config config in
+          Store.run_chains config);
     };
     {
       mode = `Chains;
       speed = `Slow;
       run =
         (fun config ->
-          Bench_inodes_2.run_chains
-            { config with inode_config = `Entries_2; store_type = `Pack });
+          let config =
+            { config with inode_config = (2, 5); store_type = `Pack }
```

Comment:
I don't think we are particularly attached to these values. I would remove these defaults from the bench suite.